File: go/p1/p1_v1.go
package main

import (
	"fmt"
	"time"
)

func problem1() int {
	result := 0

	for i := range 1000 {
		if i%3 == 0 || i%5 == 0 {
			result += i
		}
	}
	return result
}

func main() {
	start := time.Now()
	fmt.Println("Result:", problem1())
	elapsed := time.Since(start)
	fmt.Printf("Execution time: %.6f seconds\n", elapsed.Seconds())
}

File: python/p1/p1_v1.py
def problem1():
    return sum([i
                for i in range(1000)
                if (i % 3 == 0 or i % 5 == 0)])

if __name__ == "__main__":
    import time
    
    start_time = time.time()
    result = problem1()
    end_time = time.time()

    print(f"Result: {result}")
    print(f"Execution time: {end_time - start_time:.6f} seconds")